electron/js/preload.js

```diff
@@ -34,6 +34,16 @@ function subscribeToMainProcessEvents() {
   });
 }
 
+function setupIpcInterface() {
+  window.sendBadgeCount = (count) => {
+    ipcRenderer.send('badge-count', count);
+  };
+
+  window.sendDeleteAccount = (accountID, sessionID) => {
+    ipcRenderer.send('should-delete-account-data', accountID, sessionID);
```

Comment:
Can we keep the naming a little shorter? No need for should. 'delete-account-data'.